app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -365,9 +365,13 @@ def api_version
       @api_version ||= client.api_version
     end
 
-    def ca_cert_store(cert)
-      return if cert.blank?
-      OpenSSL::X509::Store.new.add_cert(OpenSSL::X509::Certificate.new(cert))
+    def ca_cert_store(certs)
+      return if certs.blank?
+      store = OpenSSL::X509::Store.new
+      certs.split(/(?=-----BEGIN)/).each do |cert|
```

Comment:
I not sure I understand the question. It's there to split based on expression, but still include the match inside the split.